### PR TITLE
refactor: renamed base to foundations and core to components

### PR DESCRIPTION
**Pull request by nmerget**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,16 +11,17 @@
...
+    "update:base:scripts": "cpr ../base/scripts packages/foundations/scripts -o",
+    "update:base:scss": "cpr ../base/scss packages/foundations/scss -o",
+    "update:base:tokens": "cpr ../base/tokens packages/foundations/tokens -o",
+    "update:base:sd-config": "cpr ../base/style-dictionary.config.json packages/foundations/style-dictionary.config.json -o",
+    "update:base:sd": "cpr ../base/style-dictionary.js packages/foundations/style-dictionary.js -o",
     "update:base": "npm-run-all -p update:base:*",
+    "postupdate:base": "replace-in-file packages/foundations/**  --configFile=scripts/base-foundation.js && node scripts/rename-db-ui-base.js",
+    "update:core:button": "cpr ../core/source/_patterns/01-elements/buttons/button.scss packages/components/src/components/button/button.scss -o",
+    "update:core:card": "cpr ../core/source/_patterns/02-components/cards/card.scss packages/components/src/components/card/card.scss -o",
+    "update:core:cards": "cpr ../core/source/_patterns/02-components/cards/cards.scss packages/components/src/components/cards/cards.scss -o",
```
Comment (mfranzke) [inline package.json:24]: why not update the script names themselves ?

Comment (nmerget) [reply package.json]: fixed

File: packages/components/mitosis.config.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const PowerAppsPlugin = require('./plugins/power-apps');
-const Debug = require('./plugins/debug');
+/* Const Debug = require('./plugins/debug'); */
```
Comment (annsch) [inline packages/components/mitosis.config.js:2]: shouldn't we delete this instead of commenting out?

Comment (nmerget) [reply packages/components/mitosis.config.js]: fixed

File: packages/components/mitosis.config.js [hunk 3]
```diff
@@ -12,12 +11,9 @@ module.exports = {
 		angular: {
 			typescript: true
 		},
-		vue3: {
```
Comment (mfranzke) [inline packages/components/mitosis.config.js:15]: what's the reason to remove vue3 ?

Comment (nmerget) [reply packages/components/mitosis.config.js:15]: we use the default settings at the moment, and don't need to overwrite it

File: scripts/base-foundation.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+	from: /@db-ui\/base/g,
+	to: '@db-ui/foundations'
+};
```
Comment (mfranzke) [inline scripts/base-foundation.js:4]: what about a comment in this file that describes what is for or what is doing (most likely some "parameters" for a replacement function).

**Review by mfranzke — APPROVED** [review]

**Pull request merged**